tests/stability/test_adaptive_scaling.py

```diff
@@ -0,0 +1,302 @@
+import time
+import uuid
+
+import dask.array as da
+import pytest
+from coiled.v2 import Cluster
+from dask import delayed
+from dask.distributed import Client, Event, wait
+
+TIMEOUT_THRESHOLD = 600  # 10 minutes
+
+
+@pytest.mark.stability
+@pytest.mark.parametrize("minimum,threshold", [(0, 300), (1, 150)])
+def test_scale_up_on_task_load(minimum, threshold):
+    """Tests that adaptive scaling reacts in a reasonable amount of time to
+    an increased task load and scales up.
+    """
+    maximum = 10
+    with Cluster(
+        name=f"test_adaptive_scaling-{uuid.uuid4().hex}",
```

Comment:
https://github.com/coiled/coiled-runtime/blob/78a8614b203971494520d273619383c08b002118/conftest.py#L249-L252

Reply:
Thanks, didn't know you had addded this in the meantime!